### PR TITLE
Octavia rsyslog - log offloading support

### PR DESCRIPTION
**Pull request by gthiemonge**

### CODE REVIEW

File: pkg/amphoracontrollers/daemonset.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func DaemonSet(
 	// The octavia [DEFAULT].graceful_shutdown_timeout is set accordingly
 	// TODO(gthiemonge) This setting must be updated/removed when Jobboard is
 	// re-enabled
-	terminationGracePeriodSeconds := int64(600)
+	terminationGracePeriodSeconds := int64(60)
```
Comment (gthiemonge) [inline pkg/amphoracontrollers/daemonset.go:107]: TODO: remove this change

**Comment by beagles** [comment]
Tested in local environment -  tailing the rsyslog pod does reveal information about amphora activities and info about connections to pool members.

**Pull request merged**